easybuild/easyconfigs/p/PyTorch/PyTorch-2.1.0_skip-test_indexing_duplicates.patch

```diff
@@ -0,0 +1,19 @@
+Skip test test_indexing_duplicates.
+This failure is a bit worrying, and not understood.
```

Comment:
At least including the error trace would be good for further investigation and discoverability of the need for this patch

Reply:
Stack trace added here, too.